requirements.txt

```diff
@@ -1 +1,3 @@
 Django~=2.2.2
+pre-commit==1.17.0
```

Comment:
Don't mix up runtime and test dependencies in one giant requirements file. Separate those.